### ISSUE TITLE
Programatically created Bourbon file breaks sourcemap

### ISSUE THREAD

**Issue by sbaechler** [liferay/liferay-themes-sdk#83]
Currently, the 'createBuourbonFile' hack prevents css build hooks (like postcss) from parsing the sourcemap.

When I try to read the generated CSS files with sourcemap I get the following error:

```
UnhandledPromiseRejectionWarning: Error: "/node_modules/liferay-theme-tasks/tmp/_bourbon.scss" is not in the SourceMap.
```

Maybe there is a better way to deal with deprecated Mixins.

**Comment by sbaechler** [comment]
It somehow works when I set `{ loadMaps: false }` to `sourcemaps.init()`. The generated sourcemaps reference the correct sources. However, loadMaps should work.

**Comment by jbalsas** [comment]
Closing as #87 has been merged and `8.0.0-rc.0` has been released.

@sbaechler, please let me know if you still find issues with this new version. 

Thanks!
